coral/src/app/features/components/layouts/LoadingTable.tsx

```diff
@@ -0,0 +1,53 @@
+import {
+  DataTable,
+  DataTableColumn,
+  EmptyState,
+  Skeleton,
+} from "@aivenio/aquarium";
+import { DataTableRow } from "@aivenio/aquarium/dist/src/utils/table/types";
+
+type LoadingTableColumn = {
+  headerName: string;
+  width?: number | `${number}%`;
+  headerInvisible?: boolean;
+};
+type LoadingTableProps = {
+  rowLength: number;
+  columns: LoadingTableColumn[];
+};
+
+//eslint-disable-next-line @typescript-eslint/no-explicit-any
+
+function LoadingTable({ rowLength, columns }: LoadingTableProps) {
```

Comment:
I think this might replace the existing `SkeletonTable` component, actually, as it is a little more flexible (can pass varying amoubnts of rows and columns) :o

Reply:
Yes I thought so, too. Would have to check what needs to change for that to work. I can add a follow up ticket for that.

Reply:
Created one https://github.com/aiven/klaw/issues/1432